ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -77,6 +77,10 @@ def __init__(
             reparameterize,
             condition_sigma_on_obs,
         )
+        if self.action_spec.continuous_size > 0 and self.action_spec.discrete_size > 0:
+            raise UnityPolicyException(
+                "Tensorflow does not support mixed action spaces. Please run with --torch."
```

Comment:
`--torch` is deprecated.
```suggestion
                "TensorFlow does not support mixed action spaces. Please run with the Torch framework."
```